imgix/urlbuilder.py

```diff
@@ -113,30 +115,58 @@ def create_url(self, path, params=None):
 
         return str(url_obj)
 
-    def create_srcset(self, path, params=None):
+    def create_srcset(
+            self, path, params={},
+            start=MIN_WIDTH, stop=MAX_WIDTH, tol=TOLERANCE):
```

Comment:
Out of curiosity, how do we feel about naming the argument `tolerance` rather than shortening it? I get the desire here to be brief, but do we think it would be more self-documenting to have the alternative?

Reply:
I'd say I'm up for it, but I'd invite you to think about it a little, maybe use it a bit. I originally had `tolerance` here but it just felt too long and clunky (not very ergonomic). If you try it and find you prefer typing the longer version I'm totally down to change this :)

Reply:
It was a pretty weak opinion to start, but I think ergonomics is a good enough reason to keep it as is.